Update WHSSDatlas_v3.01.jsonld

aeidi89 opened the pull request.

aeidi89 committed:
Update WHSSDatlas_v3.01.jsonld

lzehl commented on instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:
```diff
@@ -15,8 +15,8 @@
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
this should be: 
`"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},`

lzehl commented on instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:
```diff
@@ -380,20 +380,24 @@
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",
```
it is better to to state the full citation. you don't want people to search for the papers (at least state the doi)

aeidi89 committed:
Update WHSSDatlas_v3.01.jsonld

Fixed
- digitalIdentifier
- fullDocumentation
- howToCite
- relatedPublications

lzehl commented on instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:
```diff
@@ -381,7 +381,7 @@
 	},
   "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v3 (Osen KK, Imad J, Wennberg AE, Papp EA, Leergaard TB (2019) Waxholm S ...
```
I suggest markdown formatting (cf. example in your other PRs)

aeidi89 replied:
Thanks @lzehl 
Fixed now.

lzehl commented on instances/atlas/brainAtlasVersion/WHSSDatlas_v3.01.jsonld:
```diff
@@ -394,9 +394,9 @@
   "ontologyIdentifier": null,
   "otherContribution": null,
   "relatedPublication": [
```
please correct indentation (cf. other PRs)

aeidi89 replied:
Fixed.

aeidi89 committed:
Update WHSSDatlas_v3.01.jsonld

Changes made as requested.
